### ISSUE TITLE
Avoid inadvertent int-to-float conversion in MCS subsystem

### ISSUE THREAD

**Issue by rjplevin** [anthofflab/Mimi.jl#357]
Int values in scenario arguments are converted to floats, requiring type casting back to Int to use as an index.

**Comment by davidanthoff** [comment]
Do we need to fix this before we ship v0.6.0? Or can we fix this in a later version?

**Comment by rjplevin** [comment]
This turned out to not be an MCS bug, per se. It's a question of how the variable holding scenario arguments is typed (or not).

When all args are numeric, as in the FUND case, the Int years are converted to float:
```
julia> scenario_args = [
           :scen  => [1.0, 2.0],
           :years => [2010, 2020, 2030],
           :rate  => [0.015, 0.03, 0.05]
       ]
3-element Array{Pair{Symbol,Array{Float64,1}},1}:
  :scen => [1.0, 2.0]
 :years => [2010.0, 2020.0, 2030.0]
  :rate => [0.015, 0.03, 0.05]
```

The simple solution is to add "Any" before the array of pairs (could do Pair{Symbol, Any}[ ] to be more precise, but it works either way):
```
julia> scenario_args = Any[
           :scen  => [1.0, 2.0],
           :years => [2010, 2020, 2030],
           :rate  => [0.015, 0.03, 0.05]
       ]
3-element Array{Any,1}:
  :scen => [1.0, 2.0]
 :years => [2010, 2020, 2030]
  :rate => [0.015, 0.03, 0.05]
```
